test/Makefile

```diff
@@ -17,6 +17,18 @@ RUNTEST=r2r -L -o results.json
 r2r-tests: bins
 	${RUNTEST}
 
+8051: bins
```

Comment:
run this target from another make target or in the ci, so we can see if the test works before mergin

Reply:
I am 100% sure it works. When I test locally it works. I also pushed a very similar test on another repo. See: https://github.com/gogo2464/realtek-firmware-decompilation/blob/master/.github/workflows/.travis.yml#L24

Reply:
Ah yes I just mentionned the issue with another architecture.

Reply:
Well. You cant be sure if that will break tomorrow. Theres no sense in adding tests that never run